DXMainClient/Startup.cs

```diff
@@ -62,7 +62,8 @@ public void Execute()
                 thread.Start();
             }
 
-            GenerateOnlineIdAsync();
```

Comment:
Could you explain why making this method `GenerateOnlineIdAsync` synchronous again?